.codecov.yml

```diff
@@ -8,15 +8,23 @@ coverage:
   round: down
   range: "65...90"
 
+  # status:
+  #   project:
+  #     default:
+  #       threshold: 0.2 #Allow the coverage to drop by threshold%, and posting a success status.
+  #   patch:
+  #     default:
+  #       target: 0% # trial operation
+
```

Comment:
```suggestion
```

Reply:
I will try to persuade BR members to run the integration job before merge, it will be recovered when they agreed it.